cli/cli.ts

```diff
@@ -1900,6 +1900,32 @@ ${gcards.map(gcard => `[${gcard.name}](${gcard.url})`).join(',\n')}
 
 `, { encoding: "utf8" });
         }
+        const multiplayerGames = targetConfig?.multiplayer?.games;
+        for (const game of (multiplayerGames ?? [])) {
+            if (game.title) targetStrings[`{id:game-title}${game.title}`] = game.title;
```

Comment:
I'm a bit confused by the template literal. Is the entry in `targetStrings` going to look like `{id:game-title}hello-world: hello-world`?

Reply:
Taking "perfect fit" as an example [from here](https://github.com/microsoft/pxt-arcade/blob/master/targetconfig.json#L230), this would set the entry in the json blob as `"{id:game-title}Perfect Fit": "Perfect Fit"`, so that the translator has context on what the string is referring to (e.g. that it is a game-title and not an indication of how things snap together or something). This gets stripped off before rendering via https://github.com/microsoft/pxt/blob/master/pxtlib/commonutil.ts#L214, and shows up in crowdin something like this: 
![image](https://github.com/microsoft/pxt/assets/5615930/face7b9a-2251-4972-b374-f894a1127dc3)
from `field_note.ts`

Reply:
Oh, interesting! Thanks for the context!